src/NServiceBus.AcceptanceTesting/Support/ScenarioRunner.cs

```diff
@@ -126,11 +126,16 @@ static string GenerateTestTimedOutMessage(TimeSpan maxTime)
 
         async Task StartEndpoints(IEnumerable<ComponentRunner> endpoints)
         {
-            using var startTimeoutCts = CreateCancellationTokenSource(TimeSpan.FromMinutes(2));
+            using var timeoutCts = CreateCancellationTokenSource(TimeSpan.FromMinutes(2));
```

Comment:
I think we could simply use `[ConfigureAwaitOptions](https://source.dot.net/System.Private.CoreLib/R/84837b305d3c4237.html)` with `[SuppressThrowing](https://source.dot.net/#System.Private.CoreLib/src/libraries/System.Private.CoreLib/src/System/Threading/Tasks/ConfigureAwaitOptions.cs,cd9c59bb79f47ec0,references)`